src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/UpdateCcdCaseService.java

```diff
@@ -31,6 +33,17 @@ public UpdateCcdCaseService(IdamService idamService,
         this.ccdClient = ccdClient;
     }
 
+    @Retryable
+    public SscsCaseDetails updateCaseV2(Long caseId, String eventType, String summary, String description, IdamTokens idamTokens, Consumer<SscsCaseData> mutator) {
```

Comment:
@banderous should we add some logs here similar to the prior version of updateCase
        log.info("UpdateCase for caseId {} and eventType {}", caseId, eventType);